### PR TITLE
Make readme more friendly

### PR DESCRIPTION
**Pull request by cliserkad**

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -1,34 +1,24 @@
...
-[![MPL License][license-shield]][license-url]
-
-</div>
-
-<br /><br />
+<br><br>
 <div align="center">
   <a href="https://github.com/OpenCraft-Studios/OpenCraft">
-    <img src="https://github.com/OpenCraft-Studios/OpenCraft/blob/master/.github/assets/images/icon.png?raw=true" alt="" height="128" width="128" />
+    <img src=".github/assets/images/icon.png" alt="" height="128" width="128" />
```
Comment (CiroZDP) [inline readme.md:4]: Ok good

File: readme.md [hunk 2]
```diff
@@ -1,34 +1,24 @@
...
   </a>
 
   <h3 align="center">OpenCraft</h3>
 
   <p align="center">
     A roughly calculated clone of Minecraft
     <br /><br />
-    <a href="https://github.com/OpenCraft-Studios/OpenCraft/wiki">
-      <b>Explore the docs »</b>
+    <a href="doc">
```
Comment (CiroZDP) [inline readme.md:12]: I'm going to add wikis.

File: readme.md [hunk 3]
```diff
@@ -1,34 +1,24 @@
...
+      <b>Documentation »</b>
```
Comment (CiroZDP) [inline readme.md:13]: Isn't cool. The "Explore" word gives a unexplainable sensation

File: readme.md [hunk 4]
```diff
@@ -1,34 +1,24 @@
...
     </a>
     <br />
     <a href="https://github.com/OpenCraft-Studios/OpenCraft/issues/new?assignees=&labels=&projects=&template=bug_report.md&title=">
       Report Bugs
     </a>
     ·
-    <a href="#-downloading">
-      Download
+    <a href="#installing">
+      Installing
```
Comment (CiroZDP) [inline readme.md:21]: Okay

File: readme.md [hunk 5]
```diff
@@ -37,52 +27,64 @@
   </p>
 </div>
 
-<br />
```
Comment (CiroZDP) [inline readme.md:40]: Im not sure about that but approved!

File: readme.md [hunk 6]
```diff
@@ -37,52 +27,64 @@
...
+</div>
+
+## Introduction 📜
+**OpenCraft** is a game programmed in Java, it uses the same technologies (LWJGL 2 + Java 21) as Minecraft.
 
-## ✅ Getting started
-**OpenCraft** is a game programmed in Java, it has the same motor (LWJGL 2 + Java 17) that Minecraft.
+### Warning ⚠️
```
Comment (CiroZDP) [inline readme.md:43]: Better use [!WARNING]

File: readme.md [hunk 7]
```diff
@@ -37,52 +27,64 @@
...
   </summary><br />
   <p align="center">
-    <img width="49%" src="assets/images/screenshot1.png" alt="PNG"/>&nbsp;
-    <img width="49%" src="assets/images/screenshot2.png" alt="PNG"/>
+    <img width="49%" src=".github/assets/images/screenshot2.png" alt="PNG"/>
+    <img width="49%" src=".github/assets/images/screenshot3.png" alt="PNG"/>
   </p>
   <div align="center">
-    <img width="80%" src="assets/images/screenshot3.png" alt="PNG"/>
+    <img width="80%" src=".github/assets/images/screenshot1.png" alt="PNG"/>
```
Comment (CiroZDP) [inline readme.md:62]: No! That screenshot was beautiful. That's why it was bigger

File: readme.md [hunk 8]
```diff
@@ -37,52 +27,64 @@
...
+<h2 id="installing">🚀 Installing</h2>
+
+### Windows Script 👨‍💻
+Download [opencraft-windows.bat](https://github.com/OpenCraft-Studios/OpenCraft/releases/download/auto-release/opencraft-windows.bat)
+and double click it!
 
-###### Windows
-```batch
-java -cp OpenCraft.jar; net.opencraft.OpenCraft --gameDir opcraft
+### Download Jar ☕
```
Comment (CiroZDP) [inline readme.md:74]: In the `dev` branch there is a improved README that contains categories. Check it out!

File: readme.md [hunk 9]
```diff
@@ -112,16 +114,8 @@ java -cp OpenCraft.jar: net.opencraft.OpenCraft --gameDir opcraft/
 
 **The full license is at [BSL 1.0][license-url].**
 
-<br />
 
 
-## 👷‍♂️ Contributors
```
Comment (CiroZDP) [inline readme.md:118]: Idk, approved anyways.

**Review by CiroZDP — CHANGES_REQUESTED** [review]
Not approved. You changed so many things that were actually useful and good.

File: readme.md (hunk 4 above)
Comment (CiroZDP) [reply readme.md:21]: It isn't installing anyways. It is just executing the game.

File: readme.md (hunk 7 above)
Comment (CiroZDP) [reply readme.md:62]: Not approved :(

File: readme.md (hunk 8 above)
Comment (CiroZDP) [reply readme.md:74]: They can be useful for operating system sorting.

File: readme.md (hunk 6 above)
Comment (cliserkad) [reply readme.md:43]: Yes I didn't realize it's special syntax for github

File: readme.md (hunk 9 above)
Comment (cliserkad) [reply readme.md:118]: We can make a dedicated contributors file. Usernames are visible in the sidebar anyway

File: readme.md (hunk 6 above)
Comment (CiroZDP) [reply readme.md:43]: Delete that branch, Because dev has got a better readme. Build on top of that readme. You can add installing for more os!

**Pull request closed without merging**